### PR TITLE
Auth endpoint branch

### PR DESCRIPTION
**Pull request by josephyunzz**
# Description

**Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.**

Implemented sign-in endpoint (/auth get) for user sign in. Configured ApiDocJs and published the ApiDoc endpoint (/doc) to include documentation on how to interact with the API. Using ApiDocJs requires installation to the Node.js framework which generates a bunch of configuration files which I've already done and pushed to this branch.

Fixes [#69](https://github.com/gtullio12/Team-1-TCSS-450/projects/2#card-59897936) (issue)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Sent a get request to the /auth endpoint using postman with valid authorization headers. Expected and got a 201 success response.
- [x] Sent a get request to the /auth endpoint using postman with invalid authorization headers. Expected and got a 400 error response.
- [x] Tested /doc endpoint by running heroku on local server and hitting that endpoint.

### CODE REVIEW

**Comment by josephyunzz** [comment]
https://github.com/gtullio12/Team-1-TCSS-450-Webservice/pull/5/commits/40241b83f28e6f528028d4a83dc887098c815394

Just removing an extra 'module router exports'.

**Comment by josephyunzz** [comment]
https://github.com/gtullio12/Team-1-TCSS-450-Webservice/pull/5/commits/40241b83f28e6f528028d4a83dc887098c815394

All these files were automatically generated when installing ApiDocJs

File: routes/auth.js [hunk 1]
```diff
@@ -219,4 +219,4 @@ router.post('/', (request, response) => {
         })
 })
 
-module.exports = router;
+module.exports = router;
```
Comment (josephyunzz) [inline routes/auth.js:222]: Removed an extra 'exports'

File: routes/auth.js [hunk 2]
```diff
@@ -90,6 +96,7 @@ router.post('/', (request, response) => {
                         message: "Email exists"
                     })
                 } else {
+                    console.log(error.stack);
```
Comment (gtullio12) [inline routes/auth.js:99]: Why was this added?
